openhands/agenthub/visualbrowsing_agent/README.md

```diff
@@ -0,0 +1,7 @@
+# Browsing Agent Framework
+
+This folder implements the AgentLab [generic agent](https://github.com/ServiceNow/AgentLab/tree/main/src/agentlab/agents/generic_agent) that enables full-featured web browsing. The observations given to the agent include set-of-marks annotated web-page screenshot, accessibility tree of the web-page and all the thoughts and actions from previous steps.
+
+## Test run
+
+Note that for browsing tasks, GPT-4/Claude is usually a requirement to get reasonable results, due to the complexity of the web page structures. This agent has been evaluated on the VisualWebArena benchmark and the CodeAct agent still uses BrowsingAgent for delegation.
```

Comment:
> CodeAct agent still uses BrowsingAgent for delegation.

I believe CodeAct right now performs browsing by itself (e.g., via browse_url and browser tool). Maybe you can updated this readme.